docs/development_guide/index.rst

```diff
@@ -1,4 +1,4 @@
-Contributor's guide
+Developer's guide
 ###################
```

Comment:
Disagree with these. Contributors could (in the future) include designers, documentation writers, translators, QA people and bug reporting, etc etc. The contributor's guide would be the one place to list how-tos for all of these